nucleus/model_run.py

```diff
@@ -58,20 +59,18 @@ def commit(self, payload: Optional[dict] = None) -> dict:
             payload = {}
         return self._client.commit_model_run(self.model_run_id, payload)
 
-    def predict(self, payload: dict) -> dict:
+    def predict(self, annotations: List[Any]) -> dict:
```

Comment:
@srikanth-scale FYI for docs, updated this method to take list of 2DBoxPredictions instead of raw JSON payload.

Reply:
`annotations: List[Union[Box2DPrediction, PolygonPrediction]]`